src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -660,7 +661,7 @@ public ListenableFuture<Iterable<Digest>> findMissingBlobs(
     // risk of returning expired workers despite filtering by active workers below. This is because
     // the strategy may return workers that have expired in the last 30 seconds. However, checking
     // workers directly is not a guarantee either since workers could leave the cluster after being
-    // queried. Ultimitely, it will come down to the client's resiliency if the backplane is
+    // queried. Ultimately, it will come down to the client's resiliency if the backplane is
     // out-of-date and the server lies about which blobs are actually present. We provide this
     // alternative strategy for calculating missing blobs.
 
```

Comment:
This needs a refactor, whether before or after landing, to properly partition FMB into backplane and worker segments for clarity. If it's to be after, please follow up with an issue.

Reply:
Created followup task, I will work on this next. https://github.com/bazelbuild/bazel-buildfarm/issues/1393